common/app/common/Edition.scala

```diff
@@ -90,13 +119,30 @@ object Editionalise {
 
 }
 
-case class InternationalEdition(variant: String)
+case class InternationalEdition(variant: String) {
+  def isControl = variant == "control"
+
+  def isInternational = !isControl
+}
 
 object InternationalEdition {
+  val international = InternationalEdition("international")
+
   private val variants = Seq("control", "international")
 
-  def apply(request: RequestHeader): Option[InternationalEdition] = request.headers.get("X-GU-International")
-    .filter(variants.contains)
-    .map(InternationalEdition(_))
+  def apply(request: RequestHeader): Option[InternationalEdition] = {
+    if (Switches.InternationalEditionSwitch.isSwitchedOn) {
+      val fromHeader = request.headers.get("X-GU-International").map(_.toLowerCase)
+
+      val fromCookie = request.cookies.get("GU_EDITION").map(_.value.toLowerCase)
```

Comment:
Cookies don't work in PROD (they get wiped at the router).

We probably want to add the `X-GU-Edition` header here...